### PR TITLE
Release 5.0.0-alpha-1

### PR DESCRIPTION
**Pull request by SonataCI**
This PR was created automatically by the `sonata-project/dev-kit` project.
Make sure to manually replace the `@deprecated` comments with the appropriate version before merging.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
...
 
+## [5.0.0-alpha-1](https://github.com/sonata-project/SonataBlockBundle/compare/4.x...5.0.0-alpha-1) - 2023-06-02
+### Added
+- [[#1159](https://github.com/sonata-project/SonataBlockBundle/pull/1159)] Add return types to `FormMapper`. ([@jordisala1991](https://github.com/jordisala1991))
+
+### Changed
+- [[#1054](https://github.com/sonata-project/SonataBlockBundle/pull/1054)] Block services tagged with `sonata.block` are not made public anymore ([@dmaicher](https://github.com/dmaicher))
+
+### Removed
+- [[#1139](https://github.com/sonata-project/SonataBlockBundle/pull/1139)] Drop support for Symfony 6.0 and 6.1. ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (jordisala1991) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+- [[#1158](https://github.com/sonata-project/SonataBlockBundle/pull/1158)] Remove block serializer configuration ([@jordisala1991](https://github.com/jordisala1991))
+- [[#1064](https://github.com/sonata-project/SonataBlockBundle/pull/1064)] Drop support for PHP 7. ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (jordisala1991) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,22 @@
...
+- [[#1060](https://github.com/sonata-project/SonataBlockBundle/pull/1060)] Remove support for Symfony ^4.4 and 5.3.x ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (jordisala1991) [inline CHANGELOG.md:16]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,22 @@
...
+- [[#1060](https://github.com/sonata-project/SonataBlockBundle/pull/1060)] Remove support for Twig ^2 ([@jordisala1991](https://github.com/jordisala1991))
```
Comment (jordisala1991) [inline CHANGELOG.md:17]:
```suggestion
```

**Comment by jordisala1991** [comment]
Some of the changelog is already applied to 4.x

**Review by VincentLanglet — APPROVED** [review]

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**